libphobos/libdruntime/core/internal/convert.d

```diff
@@ -35,13 +35,30 @@ const(ubyte)[] toUbyte(T)(ref T val) if(is(Unqual!T == float) || is(Unqual!T ==
         size_t off_bytes = 0;
         size_t off_bits  = 0;
 
-        for(; off_bytes < FloatTraits!T.MANTISSA/8; ++off_bytes)
+        // Quadruples won't fit in one ulong, so check for that.
```

Comment:
You probably want to take the implementation from master, rather than my version from the riscv patches.

Reply:
OK, uses the master implementation now.